install_config/cluster_metrics.adoc

```diff
@@ -544,3 +544,19 @@ For more information about Heapster and how to access its APIs, please refer the
 link:https://github.com/kubernetes/heapster/[Heapster] project.
 
 endif::[]
+
+== Cleanup
+You can remove your metrics deployment by doing:
+[options="nowrap"]
+----
+$ oc project openshift-infra
+$ for i in $(oc get secret | egrep "(hawkular|heapster|metrics)" | awk '{ print $1 }'); do
+    oc delete secret $i
+  done
+$ oc delete rc hawkular-metrics heapster hawkular-cassandra-1
+$ oc delete svc hawkular-cassandra hawkular-cassandra-nodes hawkular-metrics heapster
+$ oc delete route hawkular-metrics
+$ oc delete sa cassandra hawkular heapster metrics-deployer
+$ oc delete template hawkular-cassandra-node-emptydir hawkular-cassandra-node-pv hawkular-cassandra-services hawkular-heapster hawkular-metrics hawkular-support
```

Comment:
You could show this line like the following to make it more readable at a glance:

```
$ oc delete template hawkular-cassandra-node-emptydir \
    hawkular-cassandra-node-pv \
    hawkular-cassandra-services \
    hawkular-heapster \
    hawkular-metrics hawkular-support
```